qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -608,7 +608,7 @@
     <item>
      <widget class="QComboBox" name="comboBox_5">
       <property name="sizePolicy">
-       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
```

Comment:
I figured letting the widget size itself was safer and slightly too big was better than possibly too small. You can see why I did this for just that widget in the screenshots below. The value displayed in both is 128, the default from the UI file.

### Before
![before](https://cloud.githubusercontent.com/assets/4886803/14584327/c7d9afec-03fe-11e6-8606-8f4e02b99154.png)
### After
![after](https://cloud.githubusercontent.com/assets/4886803/14584328/c90c62c4-03fe-11e6-92a8-3142bfcab372.png)